examples/post_training_quantization/torch/ssd300_vgg16/requirements.txt

```diff
@@ -1,7 +1,7 @@
-fastdownload
+fastdownload==0.0.7
+onnx==1.13.1
 openvino-dev==2023.1
+pycocotools==2.0.7
+torch==2.0.1  # ssd300_vgg16 can not be exported with 2.1.0: Unsupported: ONNX export of operator get_pool_ceil_padding
```

Comment:
Have you checked if this is a known issue in torch? If not, we should file an issue.

Reply:
Agree, I've opened a [Pytorch issue](https://github.com/pytorch/pytorch/issues/113155) and referenced it in the mentioned file

Reply:
Thanks